taskana-adapter-camunda-outbox-rest-spring-boot/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+######server-port########
+
+server.port=9090
+
+#####datasource#######
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
This datasource should not be named "spring.datasource" but more specifically like e.g. 
taskana.adapter.camunda.outbox.rest.spring.boot.datasource .